client/src/hooks/reducers/organisationReducers.js

```diff
@@ -1,26 +1,26 @@
 import {
-  CREATE_ORGANIZATION,
-  CREATE_ORGANIZATION_ERROR,
-  //CREATE_ORGANIZATION_SUCCESS,
-  FETCH_ORGANIZATION,
-  FETCH_ORGANIZATION_ERROR,
-  FETCH_ORGANIZATION_SUCCESS,
-  UPDATE_ORGANIZATION,
-  UPDATE_ORGANIZATION_ERROR,
-  UPDATE_ORGANIZATION_SUCCESS,
-} from "../actions/organizationActions";
+  CREATE_Organisation,
+  CREATE_Organisation_ERROR,
+  //CREATE_Organisation_SUCCESS,
```

Comment:
this could be removed since not being used

Reply:
ooh nice catch! thanks Sonia!